non_profit_organization/demo/website.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="website.default_website" model="website">
+        <field name="language_ids" eval="[(6, 0, [ref('base.lang_en')])]"/>
+        <field name="configurator_done" eval="True"/>
+        <field name="cart_recovery_mail_template_id" eval="False"/>
```

Comment:
No need for `language_ids`, `cart_recovery_mail_template_id`, `salesteam_id`, `default_lang_id` and `crm_default_team_id` fields as they are default fields